overload-test-scripts/mongo_overload.py

```diff
@@ -0,0 +1,48 @@
+from pymongo import MongoClient
+import threading
+import time
+
+# List for saving all threads results.
+results = []
+
+# function to query the database
+def query_database(db, query):
+  start_time = time.time()
+  # execute the query and print the results
+  result = db.codegard_cache.find(query)
+  for r in result:
+    results.append(r)
+    # print(r)
+  elapsed_time = time.time() - start_time
+  print(f"Thread ID: {threading.get_ident()}, Elapsed time: {elapsed_time} seconds")
+
+# create a connection to the MongoDB server
+conn_str = 'mongodb://localhost:27017'
```

Comment:
Where this port is used?
In the docker-compose file?

Reply:
yes